project-base/app/src/Migrations/Version20240722095556.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20240722095556 extends AbstractMigration
```

Comment:
Hi, this class should be actually in `Shopsys\FrameworkBundle\Migrations` namesace

Reply:
Moved it, thanks.